api/v1alpha1/amaltheasession_children.go

```diff
@@ -29,38 +29,14 @@ const sessionVolumeName string = prefix + "volume"
 const shmVolumeName string = prefix + "dev-shm"
 const authProxyPort int32 = 65535
 
+var sidecarsImage string = getSidecarsImage()
 var rcloneStorageClass string = getStorageClass()
 var rcloneDefaultStorage resource.Quantity = resource.MustParse("1Gi")
 
 const rcloneStorageSecretNameAnnotation = "csi-rclone.dev/secretName"
 
-// StatefulSet returns a AmaltheaSession StatefulSet object
-func (cr *AmaltheaSession) StatefulSet() (appsv1.StatefulSet, error) {
-	labels := labelsForAmaltheaSession(cr.Name)
-	replicas := int32(1)
-	if cr.Spec.Hibernated {
-		replicas = 0
-	}
-
-	session := cr.Spec.Session
+func (cr *AmaltheaSession) SessionVolumes() ([]v1.Volume, []v1.VolumeMount) {
```

Comment:
suggestion: I am a bit confused why `cr` is the abbreviation for `AmaltheaSession`, especially since in the other file (`auth_templates.go`) it is abbreviated as `session`. I think having a consistent abbreviation can help with code readability.

Reply:
It was meant to stand for "custom resource". I will just rename it to `as` across the board. For "amalthea session".